scripts/first-clone-setup-fast.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -ex
+
```

Comment:
Do we have to have **Y**et **A**nother **S**etup **S**cript?

Reply:
Can't the parts of the script be wrapped into build-setup?

Reply:
Perhaps the right thing to do is to add a `--firesim` flag to `build-setup.sh` that also inits firesim and firemarshal.

Reply:
The plan is to merge this script now as a temporary thing, then by the next release, the top level setup script will be modified to also do everything this script currently does.